spec/controllers/accounts_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+describe AccountsController, type: :controller do
+  describe "GET #index" do
+    let! (:accounts) {FactoryGirl.create_list(:account, 20)}
+    before do
+        get :index
+    end
+    it "returns http success" do
+        expect(response).to have_http_status(:success)
+    end
+    it "JSON body response contains expected account attributes" do
+      json_response = JSON.parse(response.body)
```

Comment:
The http object that wraps the response has helper methods to avoid doing this manually.
Like, for instance: `response.parsed_body`